components/controls/selectors/DateRangeSelector.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div>
+    <div>
+      <div class="row">
+        <div class="col-6">
+          <div style="position: absolute;right: -6px;top: 5px;">
+            to
+          </div>
+          <input
+            id="DateRange_StartDate"
+            v-model="selectedStartDate"
+            type="text"
+            class="form-control"
+            placeholder="Select start date"
+            :disabled="disabled"
+            onfocus="(this.type='date')"
+            onfocusout="(this.type='text')"
+            @blur="validate"
+          >
+        </div>
+        <div class="col-6">
+          <input
+            id="DateRange_EndDate"
+            v-model="selectedEndDate"
+            type="text"
+            class="form-control"
+            placeholder="Select end date"
+            :disabled="disabled"
```

Comment:
min should be set whe startDate is set
```suggestion
            :disabled="disabled"
            :min="selectedStartDate"
```

Reply:
Done!